### ISSUE TITLE
What does this mean

### ISSUE THREAD

**Issue by akomm** [ircmaxell/PhpGenerics#11]
I found [this](https://github.com/ircmaxell/PhpGenerics/blob/master/README.md#tldr). Can you please explain in more detail.

**Comment by rmrhz** [comment]
It's voodoo magic, use at your own risk.

**Comment by akomm** [comment]
I don't believe in magic, especially voodoo. Does that mean nothing bad can happen? Going to refactor all my projects to use this. What can possibly go wrong?

**Comment by malukenho** [comment]
Lots of things can go wrong. Mainly performance-wise. You should not use it, it uses php code to parse php code.

**Comment by akomm** [comment]
Sounds great. Unbelievable what PHP is capable to do!
